Clients/GUI/Modals/SetUserDataModal.razor

```diff
@@ -0,0 +1,62 @@
+﻿<EditForm EditContext="@editContext" OnValidSubmit="@SaveDataClickHandler">
+    <DataAnnotationsValidator />
+    <div class="form-group">
+        <label for="Name">Name:</label>
+        <InputText class="input-group-text" @bind-Value="user.Name" placeholder="Your name" />
+    </div>
+    <div class="form-group">
+        <label for="Surname">Surname:</label>
+        <InputText class="input-group-text" @bind-Value="user.Surname" placeholder="Your surname" />
+    </div>
+    <div class="form-group">
+        <label for="Birthday">Birthday:</label>
+        <InputDate class="input-group-text" @bind-Value="@user.DateOfBirth" />
+    </div>
+    <br />
+    <ValidationSummary />
+    <h4 style="display:@(ErrorOccured ? "block" : "hidden"); color:red">@ErrorMessage</h4>
+    <br />
+    <input type="submit" class="form-control button btn-primary" value="Save" />
+</EditForm>
+
+@code {
+    [Parameter]
+    public Guid userId { get; set; }
+    [CascadingParameter]
+    public BlazoredModalInstance BlazoredModal { get; set; }
+    public string ErrorMessage { get; set; }
+    public bool ErrorOccured { get; set; }
+    private SetUserDataViewModel user;
```

Comment:
set - it is usually property or method. I don't think that you need this name for fiels

Reply:
> set - it is usually property or method. I don't think that you need this name for fiels
Where're the fields?

Reply:
Okay, what is SetUserDataViewModel user;

Reply:
> Okay, what is SetUserDataViewModel user;

It's a view model which is used by form in gui in this modal

Reply:
Thanks